mungegithub/features/server.go

```diff
@@ -76,10 +80,17 @@ func (s *ServerFeature) Initialize(config *github.Config) error {
 
 	s.prometheus.loops = prometheus.NewCounter(prometheus.CounterOpts{
 		Name:        "mungegithub_loops",
-		Help:        "Number of loops performed by the queue",
+		Help:        "Number of loops performed by the munger",
+		ConstLabels: map[string]string{"org": config.Org, "repo": config.Project, "app": mungeopts.App},
+	})
+	s.prometheus.tokenUsage = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name:        "github_token_usage",
+		Help:        "Number of used github tokens by the munger",
```

Comment:
s/Number of used github tokens/Number of github tokens used/

Reply:
updated